Incorrect minimal cmake version requirement.

### Are you using SoftEther VPN 5.x?

- [X] Yes, I'm using SoftEther VPN 5.x, not 4.x.

### Version

5.02.5185 / c2a7aa548137dc80c6aafdc645cf4dc34e0dc764

### Component

VPN Server

### Operating system & version

Ubuntu 18.04

### Architecture or Hardware model

amd64

### Steps to reproduce

```
git checkout 5.02.5185
./configure
````

### ✔️ Expected Behavior

Successfully finished configuration.

### ❌ Actual Behavior

```
$ ./configure
---------------------------------------------------------------------
SoftEther VPN for Unix

Copyright (c) all contributors on SoftEther VPN project in GitHub.
Copyright (c) Daiyuu Nobori, SoftEther Project at University of Tsukuba, and SoftEther Corporation.

Licensed under the Apache License, Version 2.0 (the License).

Read and understand README, LICENSE and WARNING before use.
---------------------------------------------------------------------

Welcome to the corner-cutting configure script !

'rpm' executable found, CPack will generate RPM packages.

CMake Error at CMakeLists.txt:20 (cmake_policy):
  Policy "CMP0091" is not known to this version of CMake.


-- The C compiler identification is GNU 7.5.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Looking for sys/auxv.h
-- Looking for sys/auxv.h - found
-- Build date: 12/08/2024
-- Build time: 12:38:01
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.28")
-- Checking for one of the modules 'libsodium'
-- Found Curses: /usr/lib/x86_64-linux-gnu/libcurses.so
-- Found OpenSSL: /usr/lib/x86_64-linux-gnu/libcrypto.so (found version "1.1.1")
-- Looking for EVP_PKEY_get_raw_public_key
-- Looking for EVP_PKEY_get_raw_public_key - found
-- Looking for SSL_CTX_set_num_tickets
-- Looking for SSL_CTX_set_num_tickets - found
CMake Error at src/Mayaqua/CMakeLists.txt:38 (add_compile_definitions):
  Unknown CMake command "add_compile_definitions".


-- Configuring incomplete, errors occurred!
See also "/home/eugene/SoftEtherVPN/build/CMakeFiles/CMakeOutput.log".
```

### Anything else?

`add_compile_definitions` has been introduced in cmake 3.12 however the project requires  3.10.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.10)
+cmake_minimum_required(VERSION 3.15)
 
 set(BUILD_NUMBER CACHE STRING "The number of the current build.")
 

